packages/react/src/components/ComboBox/ComboBox-test.js

```diff
@@ -1431,5 +1431,52 @@ describe('ComboBox', () => {
       expect(screen.getByTestId('change-count').textContent).toBe('1');
       expect(findInputNode()).toHaveDisplayValue('Option 2');
     });
+
+
+  it('should close menu when focusing on AI label or toggletip', async () => {
+    render(<ComboBox {...mockProps} />);
+    
+    // Open the menu first
+    await openMenu();
+    assertMenuOpen(mockProps);
+    
+    // Create a mock AI label element and trigger focus event
+    const aiLabelElement = document.createElement('div');
+    aiLabelElement.className = 'cds--ai-label';
+    document.body.appendChild(aiLabelElement);
+    
+    // Simulate focus event on AI label
+    fireEvent.focus(aiLabelElement);
+    
+    // Menu should be closed
+    assertMenuClosed();
+    
+    // Clean up
+    document.body.removeChild(aiLabelElement);
+  });
+
+  it('should close menu when focusing on toggletip', async () => {
+    render(<ComboBox {...mockProps} />);
+    
+    // Open the menu first
+    await openMenu();
+    assertMenuOpen(mockProps);
+    
+    // Create a mock toggletip element and trigger focus event
+    const toggletipElement = document.createElement('div');
+    toggletipElement.className = 'cds--toggletip';
+    document.body.appendChild(toggletipElement);
```

Comment:
I'm not sure whether you've investigated this approach. Could this testing be done using the actual `ToggleTip` component instead of a mock?